### PR TITLE
notification badges go to the left

### PR DESCRIPTION
**Pull request by JSey**
…of the babble icon.

This time: CSS class style!

Re: https://github.com/gdpelican/babble/issues/25

### CODE REVIEW

**Comment by gdpelican** [comment]
I was able to get rid of the !important with a quite specific selector, like

```
.d-header .icons .unread-notifications.babble-unread-notifications
```

**Comment by JSey** [comment]
Right - I didn't poke around long enough. Thanks. Updated the branch, you can pull that directly if you like.

File: assets/stylesheets/babble.css [hunk 1]
```diff
@@ -84,3 +84,8 @@ li.babble-post .babble-post-content a.mention {
   padding: 5px;
   margin: 0;
 }
+
+.d-header .icons .unread-notifications.babble-unread-notifications {
```
Comment (gdpelican) [inline assets/stylesheets/babble.css]: Yo @JSey these class names don't match :(

Comment (JSey) [inline assets/stylesheets/babble.css]: Oh crap… I definitively need to stop using that web client for more than one-liners. Will get back to that asap…

Comment (JSey) [inline assets/stylesheets/babble.css]: Done.

**Comment by gdpelican** [comment]
Sweet, thanks @JSey

**Comment by JSteed** [comment]
Seems like they've merged the notification with the user icon in the latest beta:

screenshot-chanologynorge org 2015-08-29 00-33-31

Which leaves plenty of space for notifications on the right side of the Babble icon.

(No idea what's up with the spinning "loading"-thingamajig, though; something broke with the latest beta. Chat window doesn't open at all now.)

**Pull request merged**